styleguide/components/31-molecules/accordion/_accordion.scss

```diff
@@ -0,0 +1,77 @@
+.accordion {
+
+  .accordion--button {
```

Comment:
Maybe use the syntax `&--button`? To be consistent with each other?

Reply:
I can't repeat this for the ul.accordion and dl.accordion without creating additional layers of nesting.

Reply:
Good point. 👍